### PR TITLE
Clazy action: Add missing dependency qtdeclarative5-dev

### PR DESCRIPTION
**Pull request by uklotzde**
Follow-up of #3295 for main

Using a single line per dependency will be easier to maintain.

### CODE REVIEW

**Comment by Holzhaus** [comment]
LGTM, and thank you for fixing my bad code ;-) waiting for the other CI builds.

**Review by Holzhaus — APPROVED** [review]

**Comment by uklotzde** [comment]
Merging after approval to fix the clazy failures on main.

**Pull request merged**
